app/src/main/java/org/schabi/newpipe/download/DownloadDialog.java

```diff
@@ -515,7 +515,24 @@ protected void setupDownloadOptions() {
         videoButton.setVisibility(isVideoStreamsAvailable ? View.VISIBLE : View.GONE);
         subtitleButton.setVisibility(isSubtitleStreamsAvailable ? View.VISIBLE : View.GONE);
 
-        if (isVideoStreamsAvailable) {
+        prefs = PreferenceManager.getDefaultSharedPreferences(getContext());
+        final String defaultMedia = prefs.getString(getString(R.string.last_used_download_type),
+                    getString(R.string.last_download_type_video_key));
+
+        assert defaultMedia != null;
```

Comment:
According to the [docs](https://developer.android.com/reference/android/content/Context#getString(int)), `getString(resInt)` cannot be `null`, thus defaultMedia cannot be `null` (as long as we don't put anything stupid into the preferences, which we clearly do not). I think, the assertion can be removed.